server/comms/link.go

```diff
@@ -11,6 +11,8 @@ import (
 	"decred.org/dcrdex/dex/ws"
 )
 
+const readLimitAuthorized = 65536
```

Comment:
Use `math.MaxUint16`? OK to avoid an import though.

Reply:
This was actually quite random.  I don't want to imply there's some reason MaxUint16 is the best choice, unless the websocket reads really are limited to this.